eng/native/configureplatform.cmake

```diff
@@ -461,10 +461,12 @@ if(CLR_CMAKE_TARGET_UNIX)
     else()
         clr_unknown_arch()
     endif()
-else()
-    set(CLR_CMAKE_TARGET_WIN32 1)
 endif(CLR_CMAKE_TARGET_UNIX)
 
+if(CLR_CMAKE_TARGET_OS STREQUAL windows)
```

Comment:
It would look better like this. Does it fix the problem that you tried to fix?

Reply:
Yes, it does thanks.